wg-batch/charter.md

```diff
@@ -0,0 +1,73 @@
+# WG Batch Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Discuss and enhance the support for Batch (eg. HPC, AI/ML, data analytics) workloads
+in core Kubernetes. We want to unify the way users deploy batch workloads to improve
+portability and to simplify supportability for Kubernetes providers.
+
+### In scope
+
+- Additions to the batch API group, currently including Job and CronJob resources
+  that benefit batch use cases such as HPC, AI/ML and data analytics.
+- Primitives for job-level queueing, not limited to the k8s Job resource.
+- Primitives to control and maximize utilization of resources in fixed-size clusters (on-prem) and elastic clusters (cloud).
+- Runtime and scheduling support for specialized hardware (GPUs, NUMA, RDMA, etc.)
```

Comment:
`Multi-cluster scheduling for batch workloads.` 【For discussion】

From the feedback of my communication with some end users. The need for multi-cluster scheduling is important in large-scale reinforcement learning and simulation.The resources of a single cluster cannot meet the requirements of this type of batch job.

Reply:
I foresee multi-cluster support becoming an important aspect. However, with the purpose of keeping a narrow set of goals we can deliver, should we defer this for later? Although we could list it from the beginning so that we keep it in mind, but not necessarily prioritize it just yet. Thoughts?

Reply:
I think it is useful to mention it to also grab the attention of sig multi-cluster

Reply:
SG to me for `a narrow set of goals at beginning`

Reply:
I added a sentence related to multi-cluster.

Reply:
whatever outcome exists in the single-cluster construct should be a two-way door for additive mutli-cluster solutions